src/nl/rubensten/texifyidea/window/EquationPreviewToolWindowFactory.kt

```diff
@@ -0,0 +1,22 @@
+package nl.rubensten.texifyidea.window
+
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.*
+import com.intellij.ui.content.*
+
+/**
+ * Created by IntelliJ IDEA.
+ * User: Alexey.Chursin
+ * Date: Aug 25, 2010
+ * Time: 2:09:00 PM
+ */
+class EquationPreviewToolWindowFactory : ToolWindowFactory {
+    // Create the tool window content.
```

Comment:
Comment redundant (says the same as the method name, and is not KDoc). Remove and make sure there is 1 newline between the class header and the function.